src/data/shop-inventory-cow.js

```diff
@@ -0,0 +1,5 @@
+import { cowFeed, huggingMachine } from './items'
+
+const inventory = [cowFeed, huggingMachine]
+
+export default inventory
```

Comment:
This is a really nice pattern! I hadn't thought of adding more `ship-inventory-*.js` files like this. Great idea! 😄